### ISSUE TITLE
EPGD groundnet - frequencies

### ISSUE THREAD

**Issue by baq-86** [marimigu/custom-scenery-polish#1]
Nie wiem czemu wcześniej tego nie zauważyłem, ale trzeba by zaktualizować częstotliwości dla EPGD. Aktualnie jest:
 ```
<frequencies>
    <AWOS>12045</AWOS>
    <TOWER>11810</TOWER>
    <APPROACH>130875</APPROACH>
</frequencies>
```

Właściwe wartości:
```
<frequencies>
    <AWOS>120450</AWOS>
    <ATIS>129625</ATIS>
    <TOWER>118100</TOWER>
    <APPROACH>127275</APPROACH>
</frequencies>

```

**Comment by marimigu** [comment]
Już zmieniłem. Widzę że nadal jesteś czynnym użytkownikiem scenerii.

**Comment by baq-86** [comment]
Zgadza się, wciąż jestem aktywny, choć już nie tak często jak dawniej, jak bym chciał. Nowa praca i nowe obowiązki pochłaniają większość czasu, ale tym chętniej raz na jakiś czas robię jakieś loty, patrzę co się zmieniło itd. Dziękuję za szybkie wprowadzenie poprawek :)
